core/src/SYCL/Kokkos_SYCL_Instance.hpp

```diff
@@ -53,7 +53,7 @@ class SYCLInternal {
   void register_team_scratch_event(int scratch_pool_id, sycl::event event);
 
   uint32_t impl_get_instance_id() const;
-  static int m_syclDev;
+  int m_syclDev;
 
```

Comment:
So what's the intended meaning of that now non-static device ID?
It looks like it is uninitialized for non-default instances.

Reply:
I just reverted those changes now.